packages/valory/connections/ledger/tests/conftest.py

```diff
@@ -58,17 +52,11 @@
 
 PACKAGE_DIR = Path(__file__).parent.parent
 
-ETHEREUM_DEFAULT_LEDGER_CONFIG = {
-    "address": f"{DEFAULT_GANACHE_ADDR}:{DEFAULT_GANACHE_PORT}",
-    "chain_id": DEFAULT_GANACHE_CHAIN_ID,
-    # "denom": ETHEREUM_DEFAULT_CURRENCY_DENOM, # noqa: E800
-    # "gas_price_api_key": GAS_PRICE_API_KEY, # noqa: E800
-    "default_gas_price_strategy": "eip1559",
-    "gas_price_strategies": {
-        "gas_station": DEFAULT_GAS_STATION_STRATEGY,
-        "eip1559": DEFAULT_EIP1559_STRATEGY,
-    },
-}
+
+@pytest.fixture()
+def key_pairs() -> List[Tuple[str, str]]:
+    """Get the default key paris for hardhat."""
+    return KEY_PAIRS
```

Comment:
This is already available in aea_test_autonomy.fixture_helpers

Reply:
it's scoped to session and we require the fixture to be scoped to the function that's why I rewrote this

https://github.com/valory-xyz/open-autonomy/blob/ffbb3fa06ab8d1e9c920ef9326f0abf27fa79b5a/plugins/aea-test-autonomy/aea_test_autonomy/fixture_helpers.py#L436